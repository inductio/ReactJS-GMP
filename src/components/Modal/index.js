import React from 'react';
import '../../styles/modal.scss';

const Modal = ({onModalClose, children}) => {
    return (
        <div className="modal">
            <div className="modal__container">
                <button className="modal__close-btn" onClick={onModalClose}/>
                <div className="modal__content">
                    {children}
                </div>
            </div>
        </div>
    );
};

export default Modal;
