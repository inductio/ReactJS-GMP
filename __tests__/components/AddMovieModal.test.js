import React from 'react';
import renderer from "react-test-renderer";
import Modal from '../../src/components/Modal';

it('renders correctly', () => {
    const tree = renderer
        .create(<Modal modalType="ADD_MOVIE_TYPE"/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
