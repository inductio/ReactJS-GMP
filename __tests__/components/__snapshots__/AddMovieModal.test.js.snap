// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Add Movie Modal 1`] = `null`;
